modules/candidate_parameters/jsx/FamilyInfo.js

```diff
@@ -9,22 +13,28 @@ let FamilyInfo = React.createClass({
       isLoaded: false,
       loadedData: 0,
     };
-  },
-  componentDidMount: function() {
+    this.setFormData = this.setFormData.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.showAlertMessage = this.showAlertMessage.bind(this);
+    this.deleteFamilyMember = this.deleteFamilyMember.bind(this);
+  }
+
+  componentDidMount() {
     let that = this;
```

Comment:
If we are going to be using this sort of reference to this (although I'm not convinced that doing this is necessary), we should atleast retain internal consistency and decide between `self` or `that`.

Reply:
i think the `this`/`that`/`self` consistency issue should be left to a different PR